depthai_sdk/src/depthai_sdk/components/camera_component.py

```diff
@@ -17,7 +17,7 @@ class CameraComponent(Component):
     def __init__(self,
                  device: dai.Device,
                  pipeline: dai.Pipeline,
-                 source: Union[str, dai.CameraBoardSocket],
+                 source: dai.CameraBoardSocket,
```

Comment:
I'm not sure why `source` can't be string. What if user specify "cam_b,c"?

Reply:
It gets resolved in oak_camera.py already - `c` isn't really needed anymore, as it gets resolved afterward whether it's color/mono

Reply:
Yep, but now user can't provide type for sensor that supports both color and mono. I guess splitting `source` into two args will solve that problem, so it works.